src/cirq_iqm/devices/iqm_device.py

```diff
@@ -22,12 +22,14 @@
 
 import collections.abc as ca
 from math import pi as PI
-from typing import Optional, Union
+from typing import FrozenSet, Optional, Union
```

Comment:
```suggestion
from typing import Optional, Union
```
Builtin types support the type annotation syntax directly, e.g. `frozenset[int]`